ui/app/pages/settings/contact-list-tab/contact-list-tab.component.js

```diff
@@ -4,7 +4,6 @@ import ContactList from '../../../components/app/contact-list';
 import {
   CONTACT_ADD_ROUTE,
   CONTACT_VIEW_ROUTE,
-  CONTACT_MY_ACCOUNTS_ROUTE,
 } from '../../../helpers/constants/routes';
 import EditContact from './edit-contact';
 import AddContact from './add-contact';
```

Comment:
It looks like the `MyAccounts` component (two lines down from here) is still being imported. That entire component can be deleted - it's now unsued.

The `showingMyAccounts` prop is also unused and can be deleted